### ISSUE TITLE
Migrate data from a single org deployment of Sentry to a multi-org one

### ISSUE THREAD

**Issue by mic4ael** [getsentry/self-hosted#1403]
### Problem Statement

Is it possible to migrate data from separate single-org Sentry instances into a single, multi-org Sentry deployment? I checked `sentry export` and `sentry import` but they seem to work nicely only with single-org deployments.

**Comment by mic4ael** [comment]
The problem is that we have created (at CERN) several instances of Sentry 9 for different projects that needed it. Now, while preparing for the upgrade to Sentry 20.x we have noticed a drastic increase in the resources that a single instance of Sentry requires for proper operation. Therefore, we thought that maybe migrating all these instances into separate organizations on a single Sentry deployment would be much more efficient.

**Comment by chadwhitacre** [comment]
Hrm, your use case seems reasonable but yes, I'm not surprised that you've found the import/export commands to be designed for one-to-one. Have you scoped out what it would take to modify these commands?

**Comment by mic4ael** [comment]
> Hrm, your use case seems reasonable but yes, I'm not surprised that you've found the import/export commands to be designed for one-to-one. Have you scoped out what it would take to modify these commands?

Yup. Had a quick look at the `export` command (https://github.com/getsentry/sentry/blob/master/src/sentry/runner/commands/backup.py#L126=) and maybe one option would be to simply `exclude` `SentryOrg` model from the final export? However, does the `import` command preserve `primary keys` etc? If not, then maybe `export/import` workflow shoud suffice in the end.

**Comment by mic4ael** [comment]
@chadwhitacre what do you think? ;)

**Comment by mic4ael** [comment]
@chadwhitacre any feedback on this?

**Comment by AndrewMoyse** [comment]
My use-case is very similar in that I have several instances of Sentry and would like to combine them into one multi-org instance of Sentry.

**Comment by chadwhitacre** [comment]
In general our import/export code is under-maintained. I'm putting this on the backlog so stalebot leaves it alone. I want to get to this but can't promise when. If you all make progress keep us posted! 🐭

**Comment by Az8th** [comment]
Hello ! Any update about this feature request ? I would also be interested

**Comment by aldy505** [comment]
Hey @Az8th sadly we're still backlogging this as we're focusing on other stuff. See #3625 

Feel free to contribute to the Sentry repo to improve things if it's possible.
